rllib/env/tests/test_multi_agent_episode.py

```diff
@@ -2254,6 +2254,31 @@ def test_other_getters(self):
 
         # --- is_terminated, is_truncated ---
 
+    def test_cut_complex(self):
+        # Simple multi-agent episode, in which all agents always step.
+        episode = self._create_simple_episode(
+            [
+                {"a0": 0},
+                {"a2": 0},
+                {"a2": 1},
+                {"a2": 2},
+                {"a0": 1},
+                {"a2": 3},
+                {"a2": 4},
+
+                # <- BUT: actual cut here, b/c of hanging action of a2
```

Comment:
Can we clarify here why this action is hanging? And means "cut" `MAE.cut`?

Reply:
👍  done.